src/three-components/TextureUtils.ts

```diff
@@ -290,7 +287,8 @@ export default class TextureUtils extends EventDispatcher {
 
   gaussianBlur(
       cubeTarget: WebGLRenderTargetCube,
-      standardDeviationRadians: number = 0.04) {
+      standardDeviationRadians: number = 0.04,
+      outputEncoding: TextureEncoding|null = null): WebGLRenderTargetCube {
```

Comment:
If you just want this to be an optional third argument, you can express it as `outputEncoding?: TextureEncoding`. The "loose" equality conditionals are useful for this particular scenario only: checking if a value is `undefined` _or_ `null`. So, the check further down the page will work as written.